.github/workflows/docker_build_push.sh

```diff
@@ -26,7 +26,7 @@ if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
   LATEST_TAG="pr-${PR_NUM}"
 elif [[ "${GITHUB_EVENT_NAME}" == "release" ]]; then
   REFSPEC=$(echo "${GITHUB_REF}" | sed 's:refs/tags/::' | head -c 40)
-  LATEST_TAG="${REFSPEC}"
+  LATEST_TAG="latest-release"
```

Comment:
the github tag on Superset is `latest` if that's what you're pointing to?

Reply:
No this is just the docker tag name to publish when building a release.

Reply:
ok, got it. Sounds like there are mixed feelings on what to name this. `latest`, `latest-release` or `stable`
@villebro @craig-rueda

Reply:
but also I think you won't necessarily want to label every release as latest, esp if it's an rc candidate an update to an older release.

Reply:
ah, that's a good catch.

Reply:
I think we should aim to align the GitHub and Docker Hub tags. It will be slightly confusing if GitHub `latest` is `latest-release` or `stable` on Docker Hub, and `master` on GitHub is `latest` on Docker Hub.

Reply:
In my opinion, latest tag should point to latest released version. All docker official images also using this convention, because when you pull an image without a tag, it pulls the latest stable version tag.